src/Posting/QR.coffee

```diff
@@ -218,7 +218,7 @@ QR =
   setRandomizedFilename: (enabled) ->
     Conf['Randomize Filename'] = enabled
     QR.nodes.randomizedFilename.classList.toggle 'disabled', !enabled
-    if enabled
```

Comment:
The update on sync probably shouldn't touch the filenames of existing files at all. And a user click should be able to toggle the randomization of a file regardless of whether it's in a dumplist.